accessibility-checker.php

```diff
@@ -348,8 +348,17 @@ function edac_summary( $post_id ) {
 			$postid = $post_id;
 			$siteid = get_current_blog_id();
 
-			// phpcs:ignore WordPress.DB.DirectDatabaseQuery.DirectQuery, WordPress.DB.DirectDatabaseQuery.NoCaching  -- Using direct query for interacting with custom database, safe variable used for table name, caching not required for one time operation.
-			$rule_count = $wpdb->get_var( $wpdb->prepare( 'SELECT count(*) FROM %i where rule = %s and siteid = %d and postid = %d and ignre = %d', $table_name, $rule['slug'], $siteid, $postid, 0 ) );
+			// phpcs:ignore WordPress.DB.DirectDatabaseQuery.DirectQuery, WordPress.DB.DirectDatabaseQuery.NoCaching -- Using direct query for interacting with custom database, safe variable used for table name, caching not required for one time operation.
+			$rule_count = $wpdb->get_var(
+				$wpdb->prepare(
+					'SELECT count(*) FROM %i where rule = %s and siteid = %d and postid = %d and ignre = %d',
+					$table_name,
+					$rule['slug'],
+					$siteid,
+					$postid,
+					0
+				)
+			);
```

Comment:
The only change in queries was to split them to multiple lines so that it's easier to visually figure out the nesting levels and what the prepare does with the placeholders.